aspnetcore/includes/add-javascript-jquery/index.md

```diff
@@ -1,25 +1,91 @@
-<!-- comment out until fixed. See https://github.com/aspnet/Docs/issues/6004
 ## Call the Web API with jQuery
 
-In this section, an HTML page is added that uses jQuery to call the Web API. jQuery initiates the request and updates the page with the details from the API's response. Add an HTML page with the following:
+In this section, an HTML page is added that uses jQuery to call the Web API. jQuery initiates the request and updates the page with the details from the API's response. To Add an HTML page in ASP.NET Core, the **ApplicationBuilder** needs to be configured to use static files and to use the default files used on the web. For more information on this, see [Work with static files in ASP.NET Core](xref:fundamentals/static-files).
+
+```csharp
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            app.UseDefaultFiles();
+            app.UseStaticFiles();
+
+            app.UseMvc();
+        }
+```
+
+In this example, adding a simple `index.html` page in the wwwroot of the project is all that is needed.
+
+![Add New File](_static/add-new-file.png)
+
+![Add New Html](_static/add-html.png)
```

Comment:
Here's the syntax that works on includes
`[!INCLUDE[](~/includes/net-core-sdk-download-link.md)]`